internal/logging/logger.go

```diff
@@ -212,7 +193,14 @@ func Middleware() gin.HandlerFunc {
 			c.Request.ContentLength,
 		)
 
-		SugarLogger(c).Infow(
+		logger := SugarLogger(c)
```

Comment:
This is not very useful since `identity` is not being set so it reverts to using `logging.L`. Might as well use `logging.L` directly

Reply:
Good call, seems like there is more that can be cleaned up here. I'll do that now. 

I think longer term receiving a logger from either `gin.Context` or `context.Context` may be desirable though. It makes it much easier to test that logging is being applied correctly. We've had a few regression with these middleware, and being able to test some of them (ex: logging) are wired up correctly would be nice.